src/app/admin/product-form/product-form.component.html

```diff
@@ -87,11 +87,15 @@
   </div>
   <div class="col-md-6">
     <div class="card" style="width: 18rem;">
-      <img [src]="imageUrl.value" class="card-img-top" />
+      <img
+        [src]="product.imageUrl"
+        *ngIf="product.imageUrl"
```

Comment:
Put your *ngIf on the entire div not just the image tag

Reply:
I kind of prefer it on the image tag that way I can see the updates in the card as I type them even if I don't have an image URL entered.  Guess it's personal preference thing.

Reply:
Like put it as *ngIf="product" with that you can see all the changes as you make them still

Reply:
Oh ok, I gotcha.  Yea that seems to work similarly too.